interface/chat/tailwind.config.js

```diff
@@ -0,0 +1,140 @@
+/** @type {import('tailwindcss').Config} */
+/** @type {import('tailwindcss').Config} */
+
+
+
+// this function handles the opacity of color
+function withOpacityValue(variable) {
+  return ({ opacityValue }) => {
+    if (opacityValue === undefined) {
+      return `hsl(var(${variable}))`
+    }
+    return `hsl(var(${variable}) / ${opacityValue})`
+  }
+}
+
+
+module.exports = {
+  content: [
+    './app/**/*.{js,ts,jsx,tsx,mdx}', // Note the addition of the `app` directory.
+    './pages/**/*.{js,ts,jsx,tsx,mdx}',
+    './components/**/*.{js,ts,jsx,tsx,mdx}',
+  ],
+
+  theme: {
+    extend: {
+      colors: {
+        'primary-light': withOpacityValue('--primary-light'),
+        'primary-dark': withOpacityValue('--primary-dark'),
+        'primary-darker': withOpacityValue('--primary-darker'),
+
+        'secondary-light': withOpacityValue('--secondary-light'),
+        'secondary-dark': withOpacityValue('--secondary-dark'),
+        'secondary-darker': withOpacityValue('--secondary-darker'),
+
+        'secondary-a': withOpacityValue('--secondary-a'),
+        'secondary-b': withOpacityValue('--secondary-b'),
+        'secondary-c': withOpacityValue('--secondary-c'),
+        'secondary-d': withOpacityValue('--secondary-d'),
+
+        'neutral-light': withOpacityValue('--neutral-light'),
+        'neutral-dark': withOpacityValue('--neutral-dark'),
+
+
+        
+          // Teals
+          'teal': {
```

Comment:
These are @lsdake lbombs color palletes. @rstarmer can you record these somewhere they wont get lost such as origin/branding (or whatever makes sense to you).

Thank you!
-steve